### DISCUSSION TITLE
Stripe integration

### DISCUSSION THREAD

**Discussion by ameer-clara** [open-webui/open-webui#4776] [General]
Hello, I'm new to this community. I was wondering if anyone has attempted to add stripe to open-webui. Happy to work on this and contribute to the repo. Any advice on where to start? Thank you. @tjbck

**Comment by atgehrhardt** [comment]
This is not likely something OWUI would be interested in doing. It is unlikely to be merged. I would recommend forking if you would like to implement things such as payment mechanisms.

**Comment by vikramguptaone** [comment]
Hi @ameer-clara I am interested in developing stripe or some SaaS boiler plate integration with openweb UI. I would like to offer this as a service in my community and need to charge for GPU or basic upkeeping of the cloud cost. any suggestions please as how or what is the best way to integrate existing SaaS boiler plate code with Open WebUI
